setup.sh

```diff
@@ -0,0 +1,283 @@
+#!/bin/bash
+
+#
+# Doubtfire MacOS Development Install Script
```

Comment:
Probably don't need this header... plus it installs it on _both_ macOS and Linux.